dev/com.ibm.ws.jdbc_fat_krb5/publish/files/kdc-server/docker-entrypoint.sh

```diff
@@ -95,6 +95,9 @@ EOT
     
     echo "Creating XE/oracle Account"
     kadmin.local -q "addprinc -pw ${KRB5_PASS} XE/oracle@${KRB5_REALM}"
```

Comment:
Looks like this principle is no longer used so can likely be removed in the future.